rapids/test_rapids.py

```diff
@@ -53,33 +53,6 @@ def verify_spark_job(self):
     self.remove_test_script(self.XGBOOST_SPARK_TEST_SCRIPT_FILE_NAME,
                             instance_name)
 
-  @parameterized.parameters(("STANDARD", ["m", "w-0"], GPU_T4, None),
```

Comment:
what?  no.  We need to make the test pass, not delete the test.  It's working for me.  It takes 50+ minutes for cuda version 12.  Have I not created a PR yet?
No, I have not.  Let me push.  We'll want to abandon this for #1219 I think.

Reply:
It got pushed by mistake. This change was for my local testing.